### ISSUE TITLE
Definition for rule 'vue/dot-location' was not found

### ISSUE THREAD

**Issue by xcatliu** [vuejs/eslint-plugin-vue#899]
**Tell us about your environment**	
* **ESLint version:**	5.16.0
* **eslint-plugin-vue version:**	5.2.2
* **Node version:**	10.13.0

**Please show your full configuration:**

```js
module.exports = {
    parser: 'vue-eslint-parser',
    parserOptions: {
        parser: 'babel-eslint',
        ecmaVersion: 2019,
        sourceType: 'module',
        ecmaFeatures: {
            globalReturn: false,
            impliedStrict: true,
            jsx: true
        },
        requireConfigFile: false,
        allowImportExportEverywhere: false
    },
    plugins: ['vue'],
    env: {
        browser: true,
        node: true,
        commonjs: true,
        es6: true
    },
    root: true,
    rules: {
        'vue/dot-location': ['error', 'property'],
        'vue/keyword-spacing': [
            'error',
            {
                before: true,
                after: true
            }
        ],
        'vue/no-deprecated-scope-attribute': 'error',
        'vue/no-empty-pattern': 'error',
        'vue/script-indent': [
            'error',
            4,
            {
                switchCase: 1
            }
        ],
        'vue/v-slot-style': 'error',
        'vue/valid-v-slot': 'error'
    }
};
```

**What did you do?**	

```vue
<template>
    <div>
    </div>
</template>
```

**What did you expect to happen?**

No errors

**What actually happened?**

```
1:1  error  Definition for rule 'vue/dot-location' was not found                   vue/dot-location
  1:1  error  Definition for rule 'vue/keyword-spacing' was not found                vue/keyword-spacing
  1:1  error  Definition for rule 'vue/no-deprecated-scope-attribute' was not found  vue/no-deprecated-scope-attribute
  1:1  error  Definition for rule 'vue/no-empty-pattern' was not found               vue/no-empty-pattern
  1:1  error  Definition for rule 'vue/v-slot-style' was not found                   vue/v-slot-style
  1:1  error  Definition for rule 'vue/valid-v-slot' was not found                   vue/valid-v-slot
```

**Comment by BrandonYeager** [comment]
I don't think the last two are included in 5.2.2. However, the link to the rules are in the sidebar (but not in the table) at the following page (https://eslint.vuejs.org/rules/).

I confirmed in my project that the following rules produce the "was not found" error...
```
Definition for rule 'vue/dot-location' was not found (vue/dot-location)
Definition for rule 'vue/keyword-spacing' was not found (vue/keyword-spacing)
Definition for rule 'vue/no-deprecated-scope-attribute' was not found (vue/no-deprecated-scope-attribute)
Definition for rule 'vue/no-empty-pattern' was not found (vue/no-empty-pattern)
```

**Comment by ChocPanda** [comment]
+1 I have the same issue while I'm developing [eslint-config-xo-vue](https://github.com/ChocPanda/eslint-config-xo-vue) for the rules

```
Definition for rule vue/no-deprecated-scope-attribute was not found  vue/no-deprecated-scope-attribute
Definition for rule vue/dot-location was not found                   vue/dot-location
Definition for rule vue/keyword-spacing was not found                vue/keyword-spacing
Definition for rule vue/no-empty-pattern was not found               vue/no-empty-pattern
```

**Comment by reesewesterhoff** [comment]
I am also having the same problem.

```
Definition for rule 'vue/dot-location' was not found (vue/dot-location)
Definition for rule 'vue/keyword-spacing' was not found (vue/keyword-spacing)
Definition for rule 'vue/no-deprecated-scope-attribute' was not found (vue/no-deprecated-scope-attribute)
Definition for rule 'vue/no-empty-pattern' was not found (vue/no-empty-pattern)
```

**Comment by ota-meshi** [comment]
Hello.
The `dot-location` rule has not yet been released.
You can not use the `dot-location` rule using v5.2.2.

The same problem rules are probably not released.

**Comment by BrandonYeager** [comment]
Just checked the repository and can confirm that none of the "not found" error rules are included in 5.2.2.

Is there any way that added but not released rules could be held from the documentation until they officially get released? I think I spent at least an hour trying to figure out why the rules weren't working.

**Comment by escapedcat** [comment]
@ota-meshi looks like `dot-location` has not been released yet. I can't find it in [5.2.3](https://github.com/vuejs/eslint-plugin-vue/tree/v5.2.3/lib/rules).  I only see it in [master](https://github.com/vuejs/eslint-plugin-vue/tree/master/lib/rules). Is that correct?  

I agree with @BrandonYeager, would be great if released rules could be marked in the docs to avoid confusion.

**Comment by selfagency** [comment]
I'm getting this currently...

```
[Info  - 1:26:33 PM] ESLint library loaded from: /Users/daniel/.nodenv/versions/10.16.0/lib/node_modules/eslint/lib/api.js
[Error - 1:26:34 PM] Failed to load plugin 'vue' declared in '.eslintrc': Cannot find module 'eslint/lib/rules/array-bracket-spacing' Referenced from: /Users/daniel/code/kifa-web-v2/.eslintrc
```

**Comment by pi0** [comment]
I also got into a problem because of `no-deprecated-scope-attribute` which is documented but not released yet.  /cc @ota-meshi anything blocking next release? :D

**Comment by ota-meshi** [comment]
A new version has been released. I close this issue.

**Comment by escapedcat** [comment]
Yay!
